ghost/core/core/server/services/email-service/email-templates/partials/styles.hbs

```diff
@@ -1568,6 +1727,48 @@ img.kg-cta-image {
 /* -------------------------------------
     BUTTONS
 ------------------------------------- */
+
+{{#hasFeature "emailCustomizationAlpha"}}
+table.btn {
+    box-sizing: border-box;
+    width: auto;
+}
+table.btn td {
+    padding: 8px 16px;
+    background-color: #ffffff;
+    border-radius: 5px;
+    {{#hasFeature "emailCustomizationAlpha"}}
+    border-radius: {{buttonBorderRadius}};
+    {{/hasFeature}}
```

Comment:
We're already in the "emailCustomizationAlpha" conditional here so we don't need the extra one 🙂 

```suggestion
    border-radius: {{buttonBorderRadius}};
```